app/views/petitions/show.html.erb

```diff
@@ -12,69 +12,62 @@
   </div>
 </details>
 
-<%= content_tag :p, :class => 'flash-notice' do -%>
-  This e-petition is now closed
-<% end if @petition.closed? %>
+<% if @petition.rejected? %>
+  <%= render 'rejected_petition' %>
+<% else %>
 
-<h1>
-  <span class="heading-secondary">Petition</span>
-  <%= @petition.title %>
-</h1>
+  <%= content_tag :p, :class => 'flash-notice' do -%>
```

Comment:
Ok, will do! Any suggestion how to name this partial? I could only come with names like `view_petition`,  `petition_show`, `not_rejected_petition` - and I don't think they're good enough :thought_balloon: